_woc/Weekly_blogs.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "Text Sentiment Analysis Starter Blog"
+excerpt: "This is the inital blog post to derive a generalized interest for the project of Text Sentiment Analysis."
+toc: true
+toc_sticky: true
+toc_label: "Contents"
+
+categories:
+  - woc
+
+tags:
+  - Winter of Code
+  - Developers
+  - Developer Student Clubs
```

Comment:
Write your own tags!
It can be your college name or your fields of interests. Like NLP, Python or others.